guides/earn-grc.htm

```diff
@@ -78,7 +78,7 @@ <h5 class="text-center"> 1) Configure the Wallet </h5>
                         <figcaption class="text-center mb-4">
                             Note that your CPID and email will be different than in the image above
                         </figcaption>
-                        d) Click next and then press the advertise beacon button
+                        d) Click next and then press the advertise beacon button. The beacon will cost less than 1 GRC to send. If you don't have 1GRC, go to the previous step to see ways to acquire some for free. If you have GRC but receive an error about an insufficient balance, your transaction your GRC was received during must wait a few minutes to confirm, so try again in 10 minutes.

```

Comment:
needs a space "1GRC"

Reply:
Note that 10 confirms is 15 minutes, not 10 minutes

Reply:
Fixed.